quantum/rgb_matrix.c

```diff
@@ -466,6 +468,8 @@ void rgb_matrix_disable_noeeprom(void) {
     rgb_matrix_config.enable = 0;
 }
 
+uint8_t rgb_matrix_get_enable(void) { return rgb_matrix_config.enable; }
```

Comment:
```suggestion
bool rgb_matrix_is_enabled(void) { return rgb_matrix_config.enable; }
```
Why is `rgb_matrix_config.enable` two bits wide?

Reply:
It was set that way to support different modes/zone config.  It's not really be used anymore, but changing it could cause issues.

It could probably be removed, but That's a bit out of scope of this PR. 

For now, I think this would be better?
```suggestion
uint8_t rgb_matrix_is_enabled(void) { return rgb_matrix_config.enable; }
```

Reply:
Sounds good.